installation-usage.md

```diff
@@ -189,6 +190,12 @@ NAME   STATE
 spod   RUNNING
 ```
 
+## Configure a custom kubelet root directory
+
+You can configure a custom kubelet root directory in case your cluster is not using the default `/var/lib/kubelet` path.
+You can achieve this by setting the environment variable `KUBELET_DIR` in the operator deployment. This environment variable will
+be then set in the manager container as well as it will be propagated into the containers part of spod daemonset.
```

Comment:
Would it be better / possible to have this as part of the SPOD instance?

Reply:
The challenge is that the kubelet path is required in the non-root enabler sidecar which currently doesn't read any SPOD CRD before copying the profiles into the kubelet directory. 

It seems to be used inside of nodestatus controller as well.

The idea is to have this environment variable propagated everywhere that all components used the same custom kubelet directory including both manager and daemon instances.

Reply:
Alright, make sense. Any further thoughts on that @kubernetes-sigs/security-profiles-operator-maintainers ?

Reply:
Relying on env-vars would make it a lot easier to enable support for non symmetrical configuration across node pools (e.g. node pools with different OS and configuration). +1 from me on the approach.